main/zoe/api/Zoe-Intro.md

```diff
@@ -19,32 +19,41 @@ This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide
 To start using a contract, you need to *install* it on Zoe. This is putting the contract’s code into Zoe. Multiple contracts can be installed on Zoe, and each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code.
 
 
-Once installed on Zoe, a contract waits for an *offer* to be made. That starts a lifecycle for the offer and new contract instance of:
-1. Sending out *invites* to request counter-offers.
-2. Partie *redeeming* those invites and making a counter-offer.
-3. Determining when the offer *matches* a counter-offer.
-4. Either succeeding or failing to make the transaction. 
+Once installed on Zoe, a contract is instantiated, staring a lifecycle new contract instance of:
+1. Instantiating the contract with
+   ```js
+   const initialInviteP = zoe~.makeInstance(
+     installationHandle,
+     issuerKeywordRecord,
+     terms);
+   ```
+1. As shown above, `makeInstance` returns a promise for the initial invite, shown above as `initialInviteP`.
+1. Sometimes: obtaining the `publicAPI` from that initial invite, and then obtaining invites from that.
+1. Sending out invites to request offers (including counter-offers).
+1. Parties use those invites to make offers.
+1. Determining when an offer *matches* another offer.
+1. Either succeeding or failing to make the trade.
 
 
-An offer’s completion only affects that offer and the assets involved in it; essentially its contract instance. The contract it was made on keeps running, waiting for and processing new offers. 
+An offer’s completion only affects that offer and the assets involved in it. The contract instance it was made on keeps running, waiting for and processing new offers.
 
 
-*Offer safety* is automatically enforced for parties in a contract; if an offer succeeds and completes, all parties get what they wanted. If it fails to complete successfully for any reason, parties keep what they offered.
+*Offer safety* is automatically enforced for parties in a contract; when an offer *completes*, it can complete giving the requesting party what they wanted, or refunding what it offered.
```

Comment:
```suggestion
*Offer safety* is automatically enforced for parties in a contract; when an offer *completes*, it can give the requesting party what they wanted, or refund what they offered.
```